server/api/models.py

```diff
@@ -1,8 +1,11 @@
+import uuid
 from django.db import models
+from django.contrib.postgres.fields import JSONField
```

Comment:
are we deciding on postgres here?

Reply:
Honestly, I kind of like the idea of being able to swap it out a few different ways.  Is it possible to keep it database platform agnostic?
https://docs.djangoproject.com/en/2.1/ref/contrib/postgres/fields/#django.contrib.postgres.fields.JSONField
According to this, the JSONField used here allows for indexing... but I'm not sure that's a big deal on something as small as speed.

Reply:
@augustjohnson the way to keep it platform agnostic would be to not use any database-specific fields. Anything usable on sqlite *should* also be platform agnostic because it has such a minimal feature-set. Postgres just gives you a bunch of extra tooling. Storing the JSON as a string would be a universally available option